extensions/Worker.Extensions.Tables/src/TypeConverters/TableConverterBase.cs

```diff
@@ -30,11 +31,6 @@ protected bool CanConvert(ConverterContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            if (context.TargetType != typeof(T))
```

Comment:
Does this need to be moved into each individual converter? I'm wondering if there is a smart way we can keep this hear and have it work for the POCO scenario. Second, given POCO is a new scenario, can that be the fall back to if no other converter fits the bill? Just a thought, no worries if this is not possible.